packages/client/test/net/protocol/rlpxsender.spec.ts

```diff
@@ -43,7 +43,7 @@ describe('should receive status', async () => {
 
 describe('should receive message', async () => {
   const rlpxProtocol = { events: new EventEmitter() }
-  const sender = new RlpxSender(rlpxProtocol as Devp2pETH)
+  const sender = new RlpxSender(rlpxProtocol as never as Devp2pETH)
```

Comment:
Why do we have to cast it as `never` first now?

Reply:
I've reversed this edit.  For some reason, Typescript wasn't paying nicely with it before.